frontend/device/generic/device.lua

```diff
@@ -332,6 +341,28 @@ function Device:setIgnoreInput(enable) return true end
 -- Device specific method for toggling the GSensor
 function Device:toggleGSensor(toggle) end
 
+-- Whether or not the GSensor should be locked to the current orientation (i.e. Portrait <-> Inverted Portrait or Landscape <-> Inverted Landscape only)
+function Device:lockGSensor(toggle)
+    if not self:hasGSensor() then
+        return
+    end
+
+    if toggle and toggle == true then
+        -- Lock GSensor to current roientation
+        self.isGSensorLocked = yes
+    elseif toggle and toggle == false then
```

Comment:
Looks like this will never trigger :)
Just `if toggle == true` / `elseif toggle == false` and a `else` (for toggle == nil or anything else) should do.

Reply:
Oh, right. I was somehow convinced that `nil` or unset evaluated as `false`, too.